### PR TITLE
Release 1.2.0

### PR DESCRIPTION
**Pull request by beebls**
This adds new documentation for the mappings feature, and adjusts the workflows to push to the new release 1.2.0.

### CODE REVIEW

File: README.md
Comment (EMERALD0874) [inline README.md]:
```suggestion
  "manifest_version": 2,
```
Manifest version should be the latest.

**Review by EMERALD0874 — CHANGES_REQUESTED** [review]
Did you test the Python script to make sure repo validation still works? I think the new mappings feature may not be super cooperative.

**Comment by beebls** [comment]
I am going to attempt to work on new CI validation for the mappings feature.
Do not merge yet

**Comment by beebls** [comment]
CI has been modified, after merging we will need to check to ensure that it works and builds correctly

**Comment by beebls** [comment]
I bumped the github release the CI uses to clone the megajson, and this appears to be making the CI error out, we may have to manually clone the old themes.json before it begins to work.

**Review by EMERALD0874 — APPROVED** [review]
Ignoring the
# ***HEINOUS TYPOES IN YOUR REPOSITORY***
yeah we good

**Comment by suchmememanyskill** [comment]
Rekt

**Pull request merged**
